### PR TITLE
feedback_prac_06

### PR DESCRIPTION
**Pull request by Keegan-Cruickshank**
@vickiedge and @jpeggreg Please check this out for a peer review

### CODE REVIEW

File: prac_06/car.py [hunk 1]
```diff
@@ -0,0 +1,35 @@
+"""CP1404/CP5632 Practical - Car class example."""
+
+
+class Car:
+    """Represent a Car object."""
+
+    def __init__(self, fuel=0, name="Car"):
+        """Initialise a Car instance.
```
Comment (vickiedge) [inline prac_06/car.py]: Question? For doc-string multiple lines, I thought the quotations marks on first line  and last lines and text on in between lines??

File: prac_06/programming_language.py [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+
+def run_tests():
+    language1 = ProgrammingLanguage("Java", "Static", "Yes", 1995)
+    language2 = ProgrammingLanguage("Python", "Dynamic", "Yes", 1991)
+    print(language1.is_dynamic())
+    print(language2.is_dynamic())
+
+
+if __name__ == '__main__':
+    run_tests()
```
Comment (vickiedge) [inline prac_06/programming_language.py]: Looks good, just last line should end on empty line 25?

Comment (Keegan-Cruickshank) [reply prac_06/programming_language.py]: Fixed, Great Thanks!

File: prac_06/car.py (hunk 1 above)
Comment (Keegan-Cruickshank) [reply prac_06/car.py]: Perfect, Solved.

**Pull request merged**